web.tf

```diff
@@ -19,15 +19,15 @@ resource "aws_security_group" "acme_web" {
   }
 
 
- # HTTP access from the VPC - changed
```

Comment:
**Accurics remediation**
 
**[HIGH]: http port open to internet**
 
**Vulnerability**
http port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0